File: solutions/485.max-consecutive-ones/max-consecutive-ones.js
/**
 * @param {number[]} nums
 * @return {number}
 */
const findMaxConsecutiveOnes = nums => {
  let counter = 0
  let max = 0

  for (let i = 0; i < nums.length; i++) {
    let item = nums[i]

    if (item === 1) {
      counter++
    } else {
      counter = 0
    }

    max = Math.max(counter, max)
  }

  return max
}

File: solutions/485.max-consecutive-ones/max-consecutive-ones.py
class Solution:
    def findMaxConsecutiveOnes(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        ret = 0
        cnt = 0
        for n in nums:
            if n is 1:
                cnt += 1
                ret = max(cnt, ret)
            else:
                cnt = 0

        return ret
